src/text/text_formatter.rs

```diff
@@ -323,10 +323,7 @@ impl<'a, W: std::fmt::Write> IonValueFormatter<'a, W> {
     }
 
     pub fn format_integer(&mut self, value: &Integer) -> IonResult<()> {
-        match value {
-            Integer::I64(i) => write!(self.output, "{}", i)?,
-            Integer::BigInt(i) => write!(self.output, "{}", i)?,
-        }
+        write!(self.output, "{}", value)?;
```

Comment:
Is there an equivalent change in `raw_text_writer`? I know there was for `Timestamp`, but I don't have my IDE open so just checking.

Reply:
That's correct. I have added the change for `raw_text_writer` as well. Thanks!